Plus Emulator.csproj

```diff
@@ -29,6 +29,9 @@
 	  </Reference>
 	</ItemGroup>
 
+	<ItemGroup>

```

Comment:
Itemgroup is not needed.

Reply:
My IDE automatically did this, will remove

Reply:
Pushed